e2e-tests/affinity/compare/statefulset_hostname-proxysql-k127.yml

```diff
@@ -93,8 +93,8 @@ spec:
             - mountPath: /etc/proxysql/ssl-internal
               name: ssl-internal
         - args:
-            - /usr/bin/peer-list
-            - -on-change=/usr/bin/add_pxc_nodes.sh
+            - /opt/percona/peer-list
+            - -on-change=/opt/percona/haproxy_add_pxc_nodes.sh
```

Comment:
i don't think this is right

Reply:
[f34997c](https://github.com/percona/percona-xtradb-cluster-operator/pull/1753/commits/f34997c006081c906fd3b085f949e176fa436912)